pkg/controller/plan/kubevirt.go

```diff
@@ -766,42 +767,22 @@ func (r *KubeVirt) areOvirtPVCsReady(vm ref.Ref, step *plan.Step) (ready bool, e
 	return
 }
 
-var filesystemMode = core.PersistentVolumeFilesystem
-
-// Return storage profile access mode based on storage class
-func (r *KubeVirt) getStorageProfileModes(storageName string) (accessModes []core.PersistentVolumeAccessMode, volumeMode *core.PersistentVolumeMode, err error) {
-	storageProfileList := &cdi.StorageProfileList{}
-	err = r.Client.List(context.TODO(), storageProfileList)
+func (r *KubeVirt) getDefaultVolumeAndAccessMode(storageName string) ([]core.PersistentVolumeAccessMode, *core.PersistentVolumeMode, error) {
```

Comment:
could you please add a comment that refers to CDI's getDefaultVolumeAndAccessMode there?